datasets/inquisitive_qg/inquisitive_qg.py

```diff
@@ -0,0 +1,161 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Inquisitive Question Generation for High Level Text Comprehension"""
+
+from __future__ import absolute_import, division, print_function
+
+import itertools
+import os
+
+import datasets
+
+
+_CITATION = """\
+@InProceedings{ko2020inquisitive,
+  author    = {Ko, Wei-Jen and Chen, Te-Yuan and Huang, Yiyan and Durrett, Greg and Li, Junyi Jessy},
+  title     = {Inquisitive Question Generation for High Level Text Comprehension},
+  booktitle = {Proceedings of EMNLP},
+  year      = {2020},
+}
+"""
+
+_DESCRIPTION = """\
+A dataset of about 20k questions that are elicited from readers as they naturally read through a document sentence by sentence. \
+Compared to existing datasets, INQUISITIVE questions target more towards high-level (semantic and discourse) comprehension of text. \
+Because these questions are generated while the readers are pro-cessing the information, the questions directly communicate gaps between \
```

Comment:
```suggestion
Because these questions are generated while the readers are processing the information, the questions directly communicate gaps between \
```